libp2p/EndpointTracker.cpp

```diff
@@ -0,0 +1,71 @@
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
+
+#include "EndpointTracker.h"
+
+namespace dev
+{
+namespace p2p
+{
+
+// Register the statement about endpoint from one othe peers.
```

Comment:
You should use `///` comments for documentation.

Reply:
@chfast : I see a mix of both in our code base...should we always use ///? Presumably git clang-format won't set the right comment formatting for us?

Reply:
The `///` or `/**` are doxygen comments so we could generate documentation out of it. Which we will probably never do so this is not critical. This should not matter for clang-format at all.